pkg/gateway/gateway.go

```diff
@@ -152,7 +152,7 @@ func (n *jfsObjects) path(p ...string) string {
 	if len(p) > 0 && p[0] == n.conf.Format.Name {
 		p = p[1:]
 	}
-	return sep + minio.PathJoin(p...)
```

Comment:
After we fixed the bug in fs.Mkdir(), is this still needed?

Reply:
this change can fix path problem like this: "/path/b////" => "/path/b", but i think it doesn't metter